addons/website/static/src/scss/website.scss

```diff
@@ -1597,6 +1597,7 @@ header {
 .o_mega_menu {
     width: 100%;
     padding: 0;
+    top: unset!important;
```

Comment:
Can you add a space between `unset` and `!important`? :pray: (like the one below)

Reply:
done